### PR TITLE
Bug 1631506 - Add some more fields from crash annotations

### PR DESCRIPTION
**Pull request by wlach**
Also add a small script, extract_crash_annotation_fields for
automatically grabbing this type of information in the future.

Checklist for reviewer:

- [x] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title)
- [x] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [x] If the PR comes from a fork, trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional) and review the report posted in the comments.

For glean changes:
- [ ] Update `include/glean/CHANGELOG.md`

### CODE REVIEW

**Comment by wlach** [comment]
I wouldn't mind getting @acmiyaguchi to look at this one, since he's handled this type of case before. Would welcome @relud's opinion on some of the overly-long description fields in this schema though -- not sure how those will translate to e.g. bigquery's schema browser (some of them have e.g. newlines)

File: scripts/extract_crash_annotation_fields [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+"""
+
+import json
+import os
+import sys
+
+import yaml
+
+if len(sys.argv) != 2:
+    print("USAGE: {} <crash annotations file>")
```
Comment (acmiyaguchi) [inline scripts/extract_crash_annotation_fields:22]: This looks like it's missing something.

```suggestion
    print(f"USAGE: {sys.argv[0]} <crash annotations file>")
```

**Comment by relud** [comment]
> Would welcome @relud's opinion on some of the overly-long description fields in this schema though -- not sure how those will translate to e.g. bigquery's schema browser (some of them have e.g. newlines)

looks like the console's schema browser just renders any group of whitespace as a single space:
```js
# bq show --schema relud-17123:test.test_desc | jq .
[
  {
    "type": "STRING",
    "name": "field1",
    "description": "<levels>, Optional, present only if >0,\n\nstuff in another couple lines\nthat spans it's own two lines."
  }
]
```
image

File: schemas/telemetry/crash/crash.4.schema.json [hunk 2]
```diff
@@ -1296,18 +1300,34 @@
               "description": "<time>, Seconds since the Epoch (see also: `payload.crashTime`)",
               "type": "string"
             },
+            "DOMFissionEnabled": {
+              "description": "Set to 1 when DOM fission is enabled, and subframes are potentially loaded in a separate process.",
+              "type": "boolean"
```
Comment (acmiyaguchi) [inline schemas/telemetry/crash/crash.4.schema.json:1305]: I have reason to believe this is not the correct type, and that all types within the metadata field should be a string. Let me put together a query to confirm this.

**Comment by wlach** [comment]
Thanks @acmiyaguchi! Great suggestions and good catch on the types. I've updated both the generation script and the schemas per your suggestions

File: schemas/telemetry/crash/crash.4.schema.json [hunk 3]
```diff
@@ -1272,6 +1272,10 @@
               "description": "<size>, Windows-only, available physical memory in bytes",
               "type": "string"
             },
+            "AvailableSwapMemory": {
+              "description": "<size>, Amount of free swap space in bytes. - Under macOS, populated with the contents of\n  sysctl \"vm.swapusage\" :: xsu_avail.\n- Under Linux, populated with /proc/meminfo's SwapFree. - Not available on other platforms.",
```
Comment (wlach) [inline schemas/telemetry/crash/crash.4.schema.json:1276]: I modified these from the output to correspond with other fields of similar kind in the file.

**Pull request merged**